[docs] "What's new in Svelte" August newsletter

d3sandoval opened the pull request:
Covered the upcoming changes to `load` in the intro.

Let me know if I missed anything!

benmccann commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,112 @@
+
+There's lots to cover this month... with big changes are coming to SvelteKit's design before 1.0 can be completed. If you haven't already, check out Rich's Discussion, [Fixing `load`, and tightening up SvelteKit's design before 1.0 #5748](https://github.com/sveltejs/kit/discussions/5748).
+
+There's been lots of great discussion there over the last few days. If any of the comments resonate with you, feel free to drop an emoji on them 🤓
```
can you drop an emoji since the discussion has been locked? I can, but am an admin on the repo. I'm not sure if others can

Mlocik97 replied:
No, users can't

d3sandoval replied:
Thanks for the heads up! I wasn't aware :)

benmccann commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,112 @@
+- Custom Vite config locations are now supported ([#5705](https://github.com/sveltejs/kit/pull/5705))
+
+**Breaking changes:**
+- `svelte-kit` CLI commands are now run using the `vite` command and `vite.config.js` is required ([#5332](https://github.com/sveltejs/kit/pull/5332), [Docs](https://kit.svelte.dev/docs/project-structure#project-files-vite-config-js))
```
```suggestion
- `svelte-kit` CLI commands are now run using the `vite` command and `vite.config.js` is required ([#5332](https://github.com/sveltejs/kit/pull/5332), [Docs](https://kit.svelte.dev/docs/project-structure#project-files-vite-config-js)). This will allow first-class support with other projects in the Vite ecosystem like Vitest and Storybook
```

benmccann commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,112 @@
+
+**Breaking changes:**
+- `svelte-kit` CLI commands are now run using the `vite` command and `vite.config.js` is required ([#5332](https://github.com/sveltejs/kit/pull/5332), [Docs](https://kit.svelte.dev/docs/project-structure#project-files-vite-config-js))
+- `mode`, `prod` and `server` in `$app/env` have been replaced with a more consistent set of environment variables ([#5602](https://github.com/sveltejs/kit/pull/5602)/[#5663](https://github.com/sveltejs/kit/pull/5663), [Docs](https://kit.svelte.dev/docs/configuration#env))
```
https://github.com/sveltejs/kit/pull/5663 is it's own new feature that I'd highlight above instead of including as part of this bullet

Mlocik97 replied:
totally agree with Ben.

dummdidumm commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,112 @@
+authorURL: https://desandoval.net
+---
+
+There's lots to cover this month... with big changes are coming to SvelteKit's design before 1.0 can be completed. If you haven't already, check out Rich's Discussion, [Fixing `load`, and tightening up SvelteKit's design before 1.0 #5748](https://github.com/sveltejs/kit/discussions/5748).
```
Should we add a note that I joined Vercel to work on Svelte full time, meaning that there are now two developers doing so? (god that sounds narcissistic)

benmccann replied:
Yeah, I was thinking that's a good idea as well. Do you have a link to your tweet? It might be nice to include

Mlocik97 replied:
https://twitter.com/dummdidumm_/status/1549041206348222464

Mlocik97 commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,112 @@
+- Endpoint method names are now uppercased to match HTTP specifications (#5513, [Docs](https://kit.svelte.dev/docs/routing#endpoints))
+- `writeStatic` has been removed to align with Vite's config ([#5618](https://github.com/sveltejs/kit/pull/5618))
+- `transformPage` is now `transformPageChunk` ([#5657](https://github.com/sveltejs/kit/pull/5657), [Docs](https://kit.svelte.dev/docs/hooks#handle))
+
```
changes in environment variables???

Mlocik97 commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,112 @@
+
+## What's new in SvelteKit
+- Dynamically imported styles are now included during SSR ([#5138](https://github.com/sveltejs/kit/pull/5138))
+- Improvements to routes and prop updates to prevent unnecessary rerending ([#5654](https://github.com/sveltejs/kit/pull/5654), [#5671](https://github.com/sveltejs/kit/pull/5671))
```
`rerending` -> `rerendering`

Mlocik97 commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,112 @@
+**Apps & Sites built with Svelte**
+- [Hondo](https://www.playhondo.com/how-to-play) is a word guessing game with multiple rounds
+- [Hexapipes](https://github.com/gereleth/hexapipes) is a site for playing hexagonal pipes puzzle
+- Mail Must Move is an email made for those who want to get more done
```
link missing?

brittneypostma commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,112 @@
+authorURL: https://desandoval.net
+---
+
+There's lots to cover this month... with big changes are coming to SvelteKit's design before 1.0 can be completed. If you haven't already, check out Rich's Discussion, [Fixing `load`, and tightening up SvelteKit's design before 1.0 #5748](https://github.com/sveltejs/kit/discussions/5748).
```
This seems to be a typo also
`There's a lot to cover this month...big changes are coming to SvelteKit's design before 1.0 can be completed.`

brittneypostma commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,112 @@
+
+_Tech Demos_
+- [Bringing the best GraphQL experience to Svelte](https://www.the-guild.dev/blog/houdini-and-kitql) by The Guild
+- [Challenge: Explore Svelte by Building a Bubble Popping Game](https://prismic.io/blog/try-svelte-build-game) by Prismic
```
This was written by me, should we attribute that?

brittneypostma commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,112 @@
+- [SvelteKit using TS, and Storybook setup](https://www.youtube.com/watch?v=L4F5dSu0FcQ) by Jarrod Kane
+- [Building Apps with Svelte!](https://www.youtube.com/watch?v=prsXVk1fdW4) by Simon Grimm
+- [SvelteKit authentication, the better way - Tutorial](https://www.youtube.com/watch?v=Y98KipzwVdM) by Pilcrow
+
```
August Svelte Sirens talk.
```md
[Svelte Sirens August Talk - Testing in Svelte with Jess Sachs](https://sveltesirens.dev/event/testing-in-svelte)
```

d3sandoval commented:
Addressed feedback. Thanks y'all!

Does anyone know how the `backticks` in the title and first link will look on the svelte site? I can't get it to build locally anymore :/

benmccann commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,114 @@
+- `endpointExtensions` is now `moduleExtensions` and can be used to filter param matchers ([#5085](https://github.com/sveltejs/kit/pull/5085), [Docs](https://kit.svelte.dev/docs/configuration#moduleextensions))
+- Node 16.9 is now the minimum version for SvelteKit ([#5395](https://github.com/sveltejs/kit/pull/5395))
+- %-encoded filenames are now allowed. If you had a `%` in your route, you must now encode it with `%25` ([#5056](https://github.com/sveltejs/kit/pull/5056))
+- Endpoint method names are now uppercased to match HTTP specifications (#5513, [Docs](https://kit.svelte.dev/docs/routing#endpoints))
```
```suggestion
- Endpoint method names are now uppercased to match HTTP specifications ([#5513](https://github.com/sveltejs/kit/pull/5513), [Docs](https://kit.svelte.dev/docs/routing#endpoints))
```

benmccann commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,114 @@
+- Endpoint method names are now uppercased to match HTTP specifications (#5513, [Docs](https://kit.svelte.dev/docs/routing#endpoints))
+- `writeStatic` has been removed to align with Vite's config ([#5618](https://github.com/sveltejs/kit/pull/5618))
+- `transformPage` is now `transformPageChunk` ([#5657](https://github.com/sveltejs/kit/pull/5657), [Docs](https://kit.svelte.dev/docs/hooks#handle))
+
```
```suggestion
- The `prepare` script is no longer needed in `package.json` ([#5760](https://github.com/sveltejs/kit/pull/5760))
- `adapter-node` no longer does any compression while we wait for a [bug fix in the `compression` library](https://github.com/expressjs/compression/pull/183) ([#5560](https://github.com/sveltejs/kit/pull/5506))

```

benmccann commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,114 @@
+- Custom Vite modes are now respected in SSR builds ([#5602](https://github.com/sveltejs/kit/pull/5602))
+- Custom Vite config locations are now supported ([#5705](https://github.com/sveltejs/kit/pull/5705))
+- Private environment variables (aka "secrets") are now much more secure. Now if you accidentally import them to client-side code, you'll see an error ([#5663](https://github.com/sveltejs/kit/pull/5663), [Docs](https://kit.svelte.dev/docs/configuration#env))
+
```
```suggestion
- Using Vercel's v3 build output API in `adapter-vercel` ([#5514](https://github.com/sveltejs/kit/pull/5514))

```

benmccann commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,114 @@
+
+There's a lot to cover this month... big changes are coming to SvelteKit's design before 1.0 can be completed. If you haven't already, check out Rich's Discussion, [Fixing `load`, and tightening up SvelteKit's design before 1.0 #5748](https://github.com/sveltejs/kit/discussions/5748).
+
+Also, [@dummdidumm](https://github.com/dummdidumm) (Simon H) [has joined Vercel to work on Svelte full-time](https://twitter.com/dummdidumm_/status/1549041206348222464)! We're super excited to have another maintainer now dedicated to working on Svelte and SvelteKit.
```
```suggestion
Also, [@dummdidumm](https://github.com/dummdidumm) (Simon H) [has joined Vercel to work on Svelte full-time](https://twitter.com/dummdidumm_/status/1549041206348222464) and [@tcc-sejohnson](https://github.com/tcc-sejohnson) has joined the group of SvelteKit maintainers! We're super excited to have additional maintainers now dedicated to working on Svelte and SvelteKit and have already been noticing their impact. July was the third largest month for SvelteKit changes since its inception!
```

benmccann replied:
Shout out to @tcc-sejohnson as well!

dominikg commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,114 @@
+
+
+**Libraries, Tools & Components**
+- [PocketBase](https://github.com/pocketbase/pocketbase) is an open source Go backend with a single file
```
```suggestion
- [PocketBase](https://github.com/pocketbase/pocketbase) is an open source Go backend with a single file and a svelte [admin dashboard ui](https://github.com/pocketbase/pocketbase/tree/master/ui)
```

benmccann commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,119 @@
+---
+title: "What's new in Svelte: August 2022"
+description: "Changes to SvelteKit's `load` before 1.0, better Vite configurations, plus some new HTML and CSS properties!"
```
I'm not that familiar with the new html/css attributes, so am assuming they're not the biggest thing this month, but @dummdidumm can correct me on that if I'm wrong. Could probably mention Vite 3 instead 
```suggestion
description: "Changes to SvelteKit's `load` before 1.0 plus support for Vite 3 and `vite.config.js`!"
```

benmccann commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,119 @@
+
+_Starring the Svelte team_
+- [The Svelte Documentary is out!](https://www.svelteradio.com/episodes/the-svelte-documentary-is-out) on Svelte Radio
+- [Beginner SvelteKit](https://vercel.com/docs/beginner-sveltekit) by Vite
```
```suggestion
- [Beginner SvelteKit](https://vercel.com/docs/beginner-sveltekit) by Vercel
```

d3sandoval replied:
Steph is a Svelte Siren: https://sveltesirens.dev/about

benmccann commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,119 @@
+- [Svelte Schema Form](https://github.com/restspace/svelte-schema-form) is a form generator for JSON schema
+- [svelte-gesture](https://github.com/wobsoriano/svelte-gesture) is a library that lets you bind richer mouse and touch events to any component or view
+- [Snap Layout](https://github.com/ThaUnknown/snap-layout) is a window resizing module for PWAs
+- [svelte-adapter-bun](https://github.com/gornostay25/svelte-adapter-bun?c) is an adapter for SvelteKit apps that generates a standalone Bun server
```
I'm assuming `?c` is not needed. Didn't actually try
```suggestion
- [svelte-adapter-bun](https://github.com/gornostay25/svelte-adapter-bun) is an adapter for SvelteKit apps that generates a standalone Bun server
```

ThaUnknown commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,119 @@
+- [PowerTable](https://github.com/muonw/powertable) is a JavaScript component that turns JSON data into an interactive HTML table
+- [svelte-slides](https://github.com/rajasegar/svelte-slides) is a slide show template for Svelte using Reveal.js
+- [Svelte Theme Light](https://marketplace.visualstudio.com/items?itemName=webmaek.svelte-theme-light) is a Visual Studio Code theme based on the Svelte REPL
+
```
- [svelte-simple-store](https://www.npmjs.com/package/simple-store-svelte) A drop in replacement for svelte/store which uses setters/getters to write/read the store's value.
- [universal-title-bar](https://www.npmjs.com/package/universal-title-bar) Universal title bar module, which works the same both on electron and in PWA's and can be imported as a .svelte module or as a web component
- [snap-layout](https://www.npmjs.com/package/snap-layout) Windows 11's snap layout feature for webapps using svelte, can be imported as a svelte component or as a web component
-

d3sandoval replied:
I try to avoid more than one bullet from the same person. I think I'll combine `universal-title-bar` into the Snap Layout bullet and we can mentioned `svelte-simple-store` next month :)

ThaUnknown replied:
they are separate modules, which do different things, there were no issues listing multiple modules from a single person before

ThaUnknown commented on site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md:
```diff
@@ -0,0 +1,119 @@
+- [Svelte Chrome Storage](https://github.com/shaun-wild/svelte-chrome-storage) is a lightweight abstraction between Svelte stores and Chrome extension storage
+- [Svelte Schema Form](https://github.com/restspace/svelte-schema-form) is a form generator for JSON schema
+- [svelte-gesture](https://github.com/wobsoriano/svelte-gesture) is a library that lets you bind richer mouse and touch events to any component or view
+- [Snap Layout](https://github.com/ThaUnknown/snap-layout) is a window resizing module for PWAs
```
- [Snap Layout](https://github.com/ThaUnknown/snap-layout) Windows 11's snap layout feature for webapps using svelte, can be imported as a svelte component or as a web component

d3sandoval commented:
@benmccann or someone else with merge access, can we merge this, please? :)
